genericjpa/async-updates/src/main/java/org/hibernate/search/genericjpa/jpa/util/impl/JPAEntityManagerFactoryWrapper.java

```diff
@@ -0,0 +1,29 @@
+package org.hibernate.search.genericjpa.jpa.util.impl;
```

Comment:
yeah. If i remember correctly, Hibernate-Search core requires Java 7 and checkstyle therefore doesn't work on my code which currently requires Java 8.